pyvrp/Population.py

```diff
@@ -138,14 +132,14 @@ def select(self) -> Tuple[Individual, Individual]:
 
     def restart(self):
         """
-        Restarts the population. All individuals are removed and a new initial
-        population population is generated.
+        Restarts the population. All current individuals are removed and the
+        initial individuals used to seed the restarted population.
         """
         self._feas = SubPopulation(self._op, self._params)
         self._infeas = SubPopulation(self._op, self._params)
 
-        for _ in range(self._params.min_pop_size):
-            self.add(Individual.make_random(self._data, self._pm, self._rng))
+        for indiv in self._initial_solutions:
+            self.add(indiv)
```

Comment:
Do you know if this impacts performance? I don't really have a good intuition for this.

Reply:
I'd assume no, but it might not hurt to make sure.

Reply:
I'll run a benchmark just to be sure.